src/platform/sdl/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ endif()
 if(SDL_VERSION EQUAL "1.2" OR NOT SDL2_FOUND)
 	find_package(SDL 1.2)
 	if(SDL_FOUND)
+		set(SDL_VERSION "1.2")
 		set(SDL_VERSION "1.2" PARENT_SCOPE)
```

Comment:
Move this to the same block as the `add_definitions` below and make it `set(SDL_VERSION "${SDL_VERSION}" PARENT_SCOPE)` while you're doing this.